cmd/declaration_extraction/declaration_extraction_test.go

```diff
@@ -0,0 +1,546 @@
+package declaration_extraction_test
+
+import (
+	"errors"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/skycoin/cx/cmd/declaration_extraction"
+)
+
+func TestDeclarationExtraction_ReplaceCommentsWithWhitespaces(t *testing.T) {
+
+	tests := []struct {
+		scenario            string
+		testDir             string
+		wantCommentReplaced string
+	}{
+		{
+			scenario:            "Has comments",
+			testDir:             "./test_files/test.cx",
+			wantCommentReplaced: "./test_files/replaceCommentResult.cx",
+		},
+	}
+
+	for _, tc := range tests {
+		t.Run(tc.scenario, func(t *testing.T) {
+			srcBytes, err := os.ReadFile(tc.testDir)
+			if err != nil {
+				t.Fatal(err)
+			}
+			wantBytes, err := os.ReadFile(tc.wantCommentReplaced)
+			if err != nil {
+				t.Fatal(err)
+			}
+			commentReplaced := declaration_extraction.ReplaceCommentsWithWhitespaces(srcBytes)
```

Comment:
Can you also check if the length of srcBytes is equal to the length of commentReplaced? And the line number?